lib/pos_fee_estimator.go

```diff
@@ -339,6 +339,20 @@ func (posFeeEstimator *PoSFeeEstimator) EstimateFee(
 	pastBlocksPriorityPercentileBasisPoints uint64,
 	maxBlockSize uint64,
 ) (uint64, error) {
+	// If minFeeRateNanosPerKB is specified, we use that to compute the txn fee. Otherwise,
+	// we'll use our fee estimation logic to compute the fee rate to use. This allows the developer
+	// to override the fee estimation logic if they need to for special circumstances.
+	//
+	// Note that it may make sense to change the name of this to overrideFeeRateNanosPerKB, but we'd
+	// have to change it in a lot of places so I don't think it's worth it.
+	if minFeeRateNanosPerKB != 0 {
```

Comment:
We're going to need to change a lot of logic in deso-js and our frontend repos to NOT specify a fee in order to get the optimal fee market suggestion with this change. I'd rather us introduce a NEW value to represent this "override" fee rate, but we're down to crunch time so it's a bit annoying at this point.